frontend/src/host_orchestrator/main.go

```diff
@@ -105,7 +105,7 @@ func main() {
 	httpsPort := flag.Int("https_port", -1, "Port to listen on for HTTPS requests.")
 	tlsCertDir := flag.String("tls_cert_dir", defaultTLSCertDir, "Directory with the TLS certificate.")
 	cvdUser := flag.String("cvd_user", "", "User to execute cvd as.")
-	cvdCreationDockerTimeout := flag.Int("cvd_creation_docker_timeout", 30, "CVD creation timeout under docker")
+	cvdCreationTimeout := flag.Int("cvd_creation_timeout", 7, "CVD creation timeout")
```

Comment:
cvd_creation_timeout_secs

Reply:
Changed name and also fixed value as well.